package.json

```diff
@@ -41,7 +41,7 @@
         "react-lazyload": "^3.1.0",
         "react-leaflet": "^2.7.0",
         "react-router-dom": "^6.2.1",
-        "react-scripts": "^3.4.1",
```

Comment:
Do we want to update react-scripts right now?

Reply:
Oh I just realized I have a node-modules folder in a folder above the projects folder, which causes react-scripts to warn about dependencies. We don't need to update it right now/